collect_app/src/main/java/org/odk/collect/android/activities/GoogleSheetsUploaderActivity.java

```diff
@@ -295,20 +279,19 @@ public void progressUpdate(int progress, int total) {
 
     @Override
     protected Dialog onCreateDialog(int id) {
-        switch (id) {
-            case GOOGLE_USER_DIALOG:
-                AlertDialog.Builder gudBuilder = new AlertDialog.Builder(this);
-
-                gudBuilder.setTitle(getString(R.string.no_google_account));
-                gudBuilder.setMessage(getString(R.string.google_set_account));
-                gudBuilder.setPositiveButton(R.string.ok, new DialogInterface.OnClickListener() {
-                    @Override
-                    public void onClick(DialogInterface dialog, int which) {
-                        finish();
-                    }
-                });
-                gudBuilder.setCancelable(false);
-                return gudBuilder.create();
+        if (id == GOOGLE_USER_DIALOG) {
```

Comment:
It's more idiomatic in Android to match ids with switches so I think it's ok to leave even if there's only one branch. Annoying that Android Studio always complains about that, though. I don't feel strongly either way but wanted to highlight that I don't think we should be changing all.